src/components/common/MemosTable/AddReply/Reply.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import FormInput from '../../FormInput';
+import './Reply.css';
+
+const initialValues = {
+  content: '',
+};
+
+function ReplyInput(props) {
+  const [formValues, setFormValues] = useState(initialValues);
+
+  const onChange = e => {
+    setFormValues({
+      ...formValues,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  return props.trigger ? (
+    <div className="popup">
+      <div className="popup-inner">
+        <h1>Add a comment:</h1>
```

Comment:
Would you consider updating the verbiage from 'Add a comment' to 'Send a reply' or something similar so that it makes it clear that the message will be going to someone?